src/shared/syscheck_op.c

```diff
@@ -1517,7 +1517,8 @@ cJSON *win_perm_to_json(char *perms) {
         char *username = perm_node;
         perm_node = strchr(perm_node, '(');
         if (!perm_node) {
-            goto error;
+            mdebug1("Uncontrolled condition when parsing the username from '%s'. Skipping permission.", username);
```

Comment:
I think that in this case, as the message is the same as in line 1535, we should define a constant here https://github.com/wazuh/wazuh/blob/master/src/error_messages/debug_messages.h instead of hardcoding the message.
Use this constant also in the UTs